distributed/config.py

```diff
@@ -24,6 +24,7 @@
 aliases = {
     'allowed-failures': 'distributed.scheduler.allowed-failures',
     'bandwidth': 'distributed.scheduler.bandwidth',
+    'latency': 'distributed.scheduler.latency',
```

Comment:
This shouldn't be necessary.  These aliases are mostly just used for backwards compatibility with older versions.